packages/theme-patternfly-org/pages/global-css-variables.md

```diff
@@ -52,6 +83,22 @@ Breakpoint suffixes are used in utility classes and layouts as well as in many c
 
 PatternFly 4 styles provide a default starting point. You can use the variable system to make adjustments to that default styling. When you change one or more elements, you should package those values into a new SCSS stylesheet to replace the default styling.
 
+Overrides to PatternFly variables should be made at the `:root` level (for global variables) or at the top-level component selector (for component variables, ex: `.pf-c-button`), as these overrides will cascade down to children elements accordingly.
+
+Example:
+- Change the global primary color from `#06c` to `red`, but keep the `#06c` color as the background for primary buttons.
+  ```css
+  /* Override global primary color 100 to red */
+  :root {
+    --pf-global--primary-color--100: red;
+  }
+
+  /* Override the above override for only the primary button background color */
+  .pf-c-button {
+    --pf-c-button--m-primary--BackgroundColor: #06c;
```

Comment:
```suggestion
    --pf-c-button--m-primary--BackgroundColor: var(--pf-global--primary-color--100);
```

Reply:
@mcoker maybe this is a bad example, but if step 1 is overriding `--pf-global--primary-color--100` at the `:root` and then step 2 is assigning `--pf-global--primary-color--100` on `.pf-c-button` don't those do the same thing?  I was trying to demonstrate both overriding a global var but then making a more specific component-level change

Reply:
doh! Yes, sorry I meant to use `--pf-global--palette--blue-400` there - the original value for primary-color--100.